src/grammar.coffee

```diff
@@ -43,8 +43,11 @@ class Grammar
     @fileTypes ?= []
     @includedGrammarScopes = []
 
-    # Create last since Injections uses APIs from this class
-    @injections = new Injections(this, injections)
+    @initialInjections = injections
+    grammar = this
+    @registry.onDidUpdateGrammar -> grammar.recalculateInjections()
```

Comment:
This can be changed to `@registry.onDidUpdateGrammar => this.recalculateInjections()` to get rid of the `grammar` assignment, right?

Reply:
Should be correct yes, had forgotten about it.